modules/ROOT/pages/con_pipeline-git-resources.adoc

```diff
@@ -0,0 +1,84 @@
+[]
+= Git Resource
+
+Git resource represents a [git](https://git-scm.com/) repository, that contains
+the source code to be built by the pipeline. Adding the git resource as an input
+to a Task will clone this repository and allow the Task to perform the required
+actions on the contents of the repo.
+
+To create a git resource using the `PipelineResource` CRD:
+
+[source]
+----
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: wizzbang-git
+  namespace: default
+spec:
+  type: git
+  params:
+    - name: url
+      value: https://github.com/wizzbangcorp/wizzbang.git
+    - name: revision
+      value: master
+----
+
+The following resources can be added:
+
+* `url`: represents the location of the git repository, you can use this to
+    change the repo
+
+* `revision`: Git
+    [revision](https://git-scm.com/docs/gitrevisions#_specifying_revisions)
+    (branch, tag, commit SHA or ref) to clone. You can use this to control what
+    commit or branch is used. If a revision is not specified,
+    the resource will default to `latest` from `master`.
```

Comment:
Same link issue here. It should be:
Git link:https://git-scm.com/docs/gitrevisions#_specifying_revisions[revision] (branch, tag, commit SHA or ref) to clone.